Demo/src/main/java/com/braintreepayments/demo/Settings.java

```diff
@@ -121,11 +121,7 @@ public static String getAndroidPayCurrency(Context context) {
     }
 
     public static String[] getAndroidPayAllowedCountriesForShipping(Context context) {
-        String[] countries = getPreferences(context).getString("android_pay_allowed_countries_for_shipping", "US").split(",");
-        for(int i = 0; i < countries.length; i++) {
-            countries[i] = countries[i].trim();
-        }
-
+        String[] countries = getPreferences(context).getString("android_pay_allowed_countries_for_shipping", "US").split("\\s*,\\s*");
```

Comment:
although for edge case (space at the beginning and end ) it should be

String[] countries = getPreferences(context).getString("android_pay_allowed_countries_for_shipping", "US").trim().split("\\s*,\\s*");